app/utils/email-tools.js

```diff
@@ -2,11 +2,20 @@
 
 const rfc822Validator = require('rfc822-validate')
 
-module.exports = (email) => {
+function isValidEmail (email) {
```

Comment:
Do we have unit tests for this method anywhere?

Reply:
I've added some now